### DISCUSSION TITLE
Youtube videos with yt-dlp don't work on windows 7

### DISCUSSION THREAD

**Discussion by Egorkamamkinculhacker** [smplayer-dev/smplayer#642] [General]
When trying to launch a video from YouTube, it asks you to install yt-dlp. After installation, I try to run it, but it asks me to install it again. What to do?

**Comment by smplayer-dev** [comment]
I'm afraid I can't reproduce this problem in my computer with Windows 7.

**Comment by jimmy-1000** [comment]
> When trying to launch a video from YouTube, it asks you to install yt-dlp. After installation, I try to run it, but it asks me to install it again. What to do?

What is the path and folder where yt-dlp is located?

**Comment by smplayer-dev** [comment]
The location depends on the operating system. When you're about to install yt-dlp using the option `Install / Update Youtube support` from the menu `Help` it says where it will be installed.
